bookkeeper-server/src/main/java/org/apache/bookkeeper/bookie/FileInfo.java

```diff
@@ -80,6 +80,9 @@
     private int stateBits;
     private boolean needFlushHeader = false;
 
+    // lac
+    private Long lac = null;
```

Comment:
yes. it will also be used for the subsequent long poll change. so we can detect the lac changes and satisfy any long poll wait requests.